src/test/java/nextstep/subway/station/StationAcceptanceTest.java

```diff
@@ -111,5 +113,67 @@ void getStations() {
     @DisplayName("지하철역을 제거한다.")
     @Test
     void deleteStation() {
+        // given: 제거할 지하철 역을 생성한다.
+        Map<String, String> param = new HashMap<>();
+
+        param.put("name", "강남역");
+        ExtractableResponse<Response> created = insertStation(param).extract();
+        assertThat(created.statusCode()).isEqualTo(HttpStatus.CREATED.value());
```

Comment:
```suggestion
        insertStationSuccess("강남역");
```
이렇게 메서드로 분리하는건 어떻게 생각하시나요? 🤔

Reply:
오호 이러한 부분도 메소드로 분리! 메소드 명이 명확해서 오히려 가독성도 좋아지고 코드도 깔끔한거 같아요!